servers/physics/shape_sw.cpp

```diff
@@ -545,14 +545,14 @@ void CapsuleShapeSW::project_range(const Vector3 &p_normal, const Transform &p_t
 	r_min = p_normal.dot(p_transform.xform(-n));
 	return;
 
-	n = p_transform.basis.xform(n);
+	/*	n = p_transform.basis.xform(n);
```

Comment:
I don't think the tab character was intentional.

Reply:
Pressed tab instead of Enter key maybe.